matlab/matlab-client/src/matlab/PAoptions.m

```diff
@@ -621,7 +639,7 @@
         fid = fopen(optionpath, 'r');
     end
     try
-        C = textscan(fid, '%s = %[^\n]', 'CommentStyle', '%%');
+        C = textscan(fid, '%s = %[^\n\r]', 'CommentStyle', '%%');
```

Comment:
this scanf directive is supposed to eat everything up to a \n or \r character,  [^ab] means any character except a and b. 
So it should eat the content of  line, without the line ending, both in linux or windows